config/app.default.php

```diff
@@ -232,7 +239,14 @@
             'username' => 'my_app',
             'password' => 'secret',
             'database' => 'my_app',
-            'encoding' => 'utf8',
+            /*
+             * You do not need to set this flag to use full utf-8 encoding (internal default since CakePHP 3.6).
+             * If you do these are the recommendations:
+             * 'encoding' => 'utf8mb4' // On MariaDB/MySQL for full UTF-8 encoding (4-byte)
+             * 'encoding' => 'utf8mb3' // On MariaDB/MySQL for limited UTF-8 encoding (3-byte, not recommended)
```

Comment:
If it is not recommended, I wouldn't list it under the recommendations :trollface:

Reply:
If you want utf8mb3 (aka how it has been before) then that's the recommendation, but yes we can remove it.